lib/library.rb

```diff
@@ -0,0 +1,35 @@
+#require 'Book'
+require 'Date'
+
+class Library
+ attr_accessor :books  # :duedate  #TODO:status
+
+  def initialize(library)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `library`. If it's necessary, use `_` or `_library` as an argument name to indicate that it won't be used. You can also write as `initialize(*)` if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)